rice-namenode/source/main.cc

```diff
@@ -137,4 +139,14 @@ int main(int argc, char *argv[]) {
       &nncli,
       port);
   server.serve(io_service);
+
+  if (admin) {
```

Comment:
what's the reasoning behind this flag?

Reply:
@heliumj

Reply:
We are creating the admin file only for the admin

Reply:
Just pushed a new commit to fix this